rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -524,6 +502,7 @@ type CancelableProvision extends BuyerSeller: <"A data defining:  the right of a
     effectivePeriod Period (0..1) <"Effective period for cancelation when notice is given. This is the period after notice is given that cancellation becomes effecticve.">
     earliestCancellationTime BusinessCenterTime (0..1) <"The earliest time in a business day that notice of cancelation can be given.">
     latestCancelationTime BusinessCenterTime (0..1) <"The latest time at which notice of cancelation can be given.">
+    exerciseTerms ExerciseTerms (1..1)
```

Comment:
Please a brief comment for ```exerciseTerms```, like you have done for the instance under ```OptionPayout```

Reply:
Done - in #2687